pkgs/development/libraries/libpanel/default.nix

```diff
@@ -41,18 +52,23 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    "-Dinstall-examples=true"
+    (lib.mesonBool "install-examples" true)
+    (lib.mesonEnable "docs" true)
   ];
 
+  postPatch = ''
+    # generate gir even when cross compiling
+    substituteInPlace meson.build \
+      --replace-fail "not meson.is_cross_build() or" ""
```

Comment:
Can you try to upstream something like https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/777?

Reply:
Turns out it was already handling this case correctly, I misread the logic.